static/js/echo.js

```diff
@@ -373,6 +373,14 @@ function handleRealmFilter(pattern, matches) {
     return url;
 }
 
+function handleTex(tex, fullmatch) {
+    try {
+        return katex.renderToString(tex);
+    } catch (ex) {
+        return '<span class="tex-error">' + fullmatch + '</span>';
```

Comment:
You probably want to escape `fullmatch` to avoid XSS? (or maybe that's already escaped)